Content.Server/GameObjects/Components/Mobs/Speech/MouseAccentComponent.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Robust.Shared.GameObjects;
+using System.Collections.Generic;
+using Robust.Shared.Random;
+using Robust.Shared.IoC;
+
+namespace Content.Server.GameObjects.Components.Mobs.Speech
+{
+    [RegisterComponent]
+    public class MouseAccentComponent : Component, IAccentComponent
+    {
+        [Dependency] private readonly IRobustRandom _random = default!;
+        public override string Name => "MouseAccent";
+
+        private static readonly IReadOnlyList<string> Squeek = new List<string>{
+            "Squeak!", "Piep!", "Chuu!"
+        }.AsReadOnly();
+        private string RandomSqueek => _random.Pick(Squeek);
+
+        public string Accentuate(string message)
+        {
+            var squeak = message.Replace(message, RandomSqueek);
+            return new string(squeak);
```

Comment:
just return randomsqueek

swept no offense but this is actually vile code

Reply:
Done